tcf_website/static/common/sorting.js

```diff
@@ -20,6 +20,38 @@ function cmpByProp(prop, asc) {
         if (prop === "date") { // convert date string to int
             valA = dateToInt(valA);
             valB = dateToInt(valB);
+        } else if (prop === "recency") {
+            try {

```

Comment:
Since you use the same regex parsing code for `valA` and `valB`, I'd define a helper function you can call twice so you don't repeat code.

Reply:
Sounds good, I added getSessionNum() to fix this.